package.json

```diff
@@ -7,7 +7,7 @@
     "postinstall": "./postinstall.sh"
   },
   "devDependencies": {
-    "rn-nodeify": "^8.2.0"
+    "rn-nodeify": "8.2.0"
```

Comment:
I removed all the ^ so that nothing upgrades without us explicitely changing it even after deleting package-lock. Some transitive dependencies will still update and we might want to pin there here as well if we want to vendor in everything someday

Reply:
nice catch